hudi-common/src/main/java/org/apache/hudi/metrics/Metrics.java

```diff
@@ -166,6 +169,17 @@ public void registerGauge(String metricName, final long value) {
     }
   }
 
+  public HoodieGauge<Long> registerGauge(String metricName) {
+    try {
```

Comment:
why can't we call the other method here. 

```
registerGauge(String metricName, 0L); 
```

Reply:
lets address this minor comment and we are good to land

Reply:
Addressed